<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playlistItemOverlay_Widget</class>
 <widget class="QWidget" name="playlistItemOverlay_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="topVBoxLayout" columnstretch="0,1">
   <item row="1" column="1">
    <widget class="QSpinBox" name="alignmentHozizontal"/>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="alignmentMode"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Offset X</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="alignmentVertical"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Offset Y</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Alignment</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>alignmentMode</tabstop>
  <tabstop>alignmentHozizontal</tabstop>
  <tabstop>alignmentVertical</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
